ChineseFilter/sections/antiadblock.txt

```diff
@@ -1,5 +1,11 @@
 !
-! These rules are against all kinds of ad blocking reinjection/circumvention scripts
+! First of all, this section has common rules that can help prevent most known detections. In that case domain should be added to the common rules.
```

Comment:
` that can help prevent most known detections.` => ` that prevent most known ad block detections.` or something like that.